README.md

```diff
@@ -26,74 +27,90 @@ import (
     "fmt"
     "os"
 
-    "github.com/parg"
+    "github.com/clagraff/argparse"
 )
 
 func main() {
-    p := parg.NewParser("Output text based on user input")
-    p.AddHelp() // Enable `--help` & `-h` to output help text
+    p := argparse.NewParser("Output text based on user input").Version("1.3.0a")
+    p.AddHelp().AddVersion() // Enable help and version flags
 
+    dry_run := argparse.NewFlag("n dry-run", "dry", "Enable dry-run mode").Default("false")
+    max := argparse.NewArg("m max", "max", "Max number of widgets").Default("0")
+
+    p.AddOptions(dry_run, max)
 
     // Parse all available program arguments (except for the program path).
-    if err := p.Parse(os.Args[1:]...); err != nil {
-        // An error occurred? Print it out, and display the help text!
-        fmt.Println(err)
-        p.ShowHelp()
+    
+    if ns, err := p.Parse(os.Args[1:]...); err != nil {
```

Comment:
Wow, thanks. Im actually surprised someone actually commented on this. 
Never considered someone might actually stumble onto this code. 

Thanks, @coddingtonbear  👍